Calorie Tracker/Calorie Tracker/Date.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Date.swift
+//  Calorie Tracker
+//
+//  Created by Alex Thompson on 2/25/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    static func dateToString(from date: Date) -> String {
```

Comment:
Rather than static, turn this into a regular instance method so you can call it on the date directly:
```
date.formatted
```